docs/source/traits_user_manual/defining.rst

```diff
@@ -405,6 +405,56 @@ the table.
 |                  | *allow_none* = False, *adapt* = 'yes', \*\*\ *metadata*])|
 +------------------+----------------------------------------------------------+
 
+.. index:: Instance trait
+
+.. _instance:
+
+Instance
+::::::::
+One of the most fundamental and useful predefined trait types is
+:class:`~.Instance`. Instance trait values are an instance of a particular class
+or its subclasses, as specified by the **klass** argument. **klass** can be
+either an instance of a class or a class itself (note this applies to all python
+classes, not necessarily just ``HasTraits`` subclasses).
```

Comment:
Suggest changing ``HasTraits`` to a reST link to the `HasTraits` class. (I know we're missing a lot of such links in existing docs, but I'd like to slowly move towards using them.)

```suggestion
classes, not necessarily just :class:`~.HasTraits` subclasses).
```